node-commons/src/test/java/org/apache/iotdb/commons/ClientManagerTest.java

```diff
@@ -408,7 +409,9 @@ public void InvalidAsyncClientReturnClientManagersTest() throws Exception {
   }
 
   public void startServer() throws IOException {
-    metaServer = new ServerSocket(9003);
+    metaServer = new ServerSocket();
+    metaServer.setReuseAddress(true);
+    metaServer.bind(new InetSocketAddress(9003));
```

Comment:
```suggestion
    // reuse the port to avoid `Bind Address already in use` which is caused by TIME_WAIT state because port won't be usable immediately after we close it.
    metaServer.setReuseAddress(true);
    metaServer.bind(new InetSocketAddress(9003));
```

Reply:
Fixed.